shapely/geometry/base.py

```diff
@@ -531,7 +531,7 @@ def _transformed_rects():
 
     def buffer(self, distance, resolution=16, quadsegs=None,
                cap_style=CAP_STYLE.round, join_style=JOIN_STYLE.round,
-               mitre_limit=5.0):
+               mitre_limit=5.0, single_side=0):
```

Comment:
My only suggestion is to add a mention of this new keyword argument in the docstring. The docstring predates my discovery of numpydoc and lacks structure, but somewhere at the end after the mentions of other arguments would be fine.

Reply:
Thanks for the quick review.

Forgot about the docstring. I didn't know about numpydoc, it was already somewhat close so I reformatted the docstring to match that style.